src/client.py

```diff
@@ -502,6 +503,19 @@ def login(input_user_name, input_private_key):
             my_private_key = input_private_key
     return my_account_id, my_private_key
 
+def handle_httperror(err):
+    LOGGER.error(err)
+    errmsg = str(err)
+    if errmsg.startswith('400 Client Error: Bad Request for url: '):
+        print('処理に失敗しました。EoAアドレスが正しく、'
+            'また操作に十分なETHERを保有していることを、'
+            'メニューの "[1]:アカウント・保有トークン情報" で'
+            '確認してください。')
```

Comment:
EOAは全て大文字表記にする。ETH/Etherと表記する。vio.printを利用して書く
```suggestion
        self.view.vio.print('Make sure that enough Ether to operation'
                'or the EOA address and private key pair are correct.')
```

Reply:
一文ぬけていました
```suggestion
        self.view.vio.print('Make sure that enough Ether to operation'
                'or the EOA address and private key pair are correct.'
                'You can check your Ether balance from the menu [1].')
```